packages/menu/src/Menu/Menu.tsx

```diff
@@ -102,6 +103,10 @@ export function Menu({
     uncontrolledSetOpen;
   const open = controlledOpen ?? uncontrolledOpen;
 
+  const closeMenu = useCallback(() => {
```

Comment:
I think in this case useCallback is actually less performant than just using setOpen: https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
Ah that's fair. I initially had written some side effects, but removed them